src/platform-includes/sourcemaps/primer/javascript.nextjs.mdx

```diff
@@ -0,0 +1,5 @@
+To show readable stack traces in your Sentry errors, once configured, the `@sentry/nextjs` SDK will generate and upload source maps automatically.
```

Comment:
```suggestion
`@sentry/nextjs` will generate and upload source maps automatically, in order to enable errors to have readable stacktraces.
```

Not wedded to this wording specifically, but as it stands the `once configured` naturally attaches itself to the first clause, not the second. I found myself having to go back and re-read the sentence, which is what I'm trying to avoid in my rewording.